From ae6efc794cda61535259630c6e46ef98d68819a1 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Mon, 20 Jun 2022 18:30:07 +0300
Subject: [PATCH] [SW-0] gaudi2: use macro for dccm size

Used macro for PPBOOT DCCM size instead of
hardcoded value in ppboot.lds.S.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: I888a44ad8bb9a07b4f0a6dcd0e2ddbd1c8eb14da
---

diff --git a/arch/gaudi2/arc/common.h b/arch/gaudi2/arc/common.h
index bbad465..7ee9f12 100644
--- a/arch/gaudi2/arc/common.h
+++ b/arch/gaudi2/arc/common.h
@@ -81,6 +81,7 @@
  * for its own BSS and Stack.
  */
 #define PPBOOT_DCCM_BASE	0x27064030
+#define PPBOOT_DCCM_SZ		0x2000	/* 8KB */
 
 #define HL_SPI_BASE		0x26000000
 #define HL_CFG_BASE		0xF0000000
diff --git a/arch/gaudi2/arc/ppboot.lds.S b/arch/gaudi2/arc/ppboot.lds.S
index 951fe4d..f9da919 100644
--- a/arch/gaudi2/arc/ppboot.lds.S
+++ b/arch/gaudi2/arc/ppboot.lds.S
@@ -13,7 +13,7 @@
 MEMORY
 {
 	.sram 	: ORIGIN = PPBOOT_TEXT_BASE, LENGTH = PPBOOT_SIZE
-	DCCM	: ORIGIN = PPBOOT_DCCM_BASE, LENGTH = 8K
+	DCCM	: ORIGIN = PPBOOT_DCCM_BASE, LENGTH = PPBOOT_DCCM_SZ
 }
 
 REGION_ALIAS("REGION_DATA",	DCCM)
